### ISSUE TITLE
NaN Loss after ~50 iterations

### ISSUE THREAD

**Issue by georgy95** [limingcv/Photorealistic-Style-Transfer#9]
I get NaN losses after a few iterations.

I tried to reduce the learning rate, without success.

I tested on a 256x256 input image and 512x512 style image using CPU, `pytorch=1.1.0`.
